src/main/java/dobby/gui/MainWindow.java

```diff
@@ -0,0 +1,70 @@
+package dobby.gui;
+
+import dobby.Dobby;
+import dobby.Ui;
+import javafx.fxml.FXML;
+import javafx.geometry.Insets;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Dobby dobby;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image dobbyImage = new Image(this.getClass().getResourceAsStream("/images/DaDobby.jpeg"));
+
+    /**
+     * Greet the user and initialise the scroll pane
+     */
+    @FXML
+    public void initialize() {
+        dialogContainer.getChildren().addAll(
+                DialogBox.getDukeDialog(Ui.greet(), dobbyImage)
+        );
+        dialogContainer.setBackground(
```

Comment:
Maybe it is better to abstract Background and BackgroundFill  into a variable, so that your code is not looking so Arrow-headed